### PR TITLE
Update release notes for 8.1.1

### PR DESCRIPTION
**Pull request by robbavey**
**PREVIEW:** https://logstash_13907.docs-preview.app.elstc.co/guide/en/logstash/current/logstash-8-1-1.html

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -12,6 +13,55 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-1-1]]
+=== Logstash 8.1.1 Release Notes
+
+==== Notable issues fixed
+
+* Uninstalling a plugin with `bin/logstash-plugin uninstall <plugin>` will no longer fail https://github.com/elastic/logstash/pull/13823[#13823]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:21]:
```suggestion
* The`bin/logstash-plugin uninstall <plugin>` command works as expected, successfully uninstalling the specified plugin  https://github.com/elastic/logstash/pull/13823[#13823]
```

Comment (karenzone) [reply docs/static/releasenotes.asciidoc:21]: Our writing guidelines encourage writing to the positive

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -12,6 +13,55 @@ This section summarizes the changes in the following releases:
...
+* Logstash CLI tools are now able to use the selected JDK on Windows https://github.com/elastic/logstash/pull/13839[#13839]
+* Logstash will no longer fail to find JVM located in a path including spaces on Windows https://github.com/elastic/logstash/pull/13881[#13881]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:23]:
```suggestion
* Logstash can successfully locate the Windows JVM, even if the path includes spaces https://github.com/elastic/logstash/pull/13881[#13881]
```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -12,6 +13,55 @@ This section summarizes the changes in the following releases:
...
+* Fix: pre-flight checks before creating DLQ reader https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/35[#35]
+* Fix: avoid Logstash crash on shutdown if DLQ files weren't created https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/33[#33]
+
+*Elasticsearch Input - 4.12.2*
+
+* Fix: hosts => "es_host:port" regression https://github.com/logstash-plugins/logstash-input-elasticsearch/pull/168[#168]
+
+*Http_poller Input - 5.2.1*
+
+* Deps: unpin rufus-scheduler dependency https://github.com/logstash-plugins/logstash-input-http_poller/pull/130[#130]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:51]: Links to an unmerged draft PR. Not sure how or why this one got picked up. Changelog and PR numbers seem to check out, so 🤷

**Review by karenzone — COMMENTED** [review]
Left some suggestions for consideration and ideation. That is, I'm not locked into suggested text, and we can iterate if you want to.

File: docs/static/releasenotes.asciidoc (hunk 3 above)
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:51]:
```suggestion
* Deps: unpin rufus-scheduler dependency https://github.com/logstash-plugins/logstash-input-http_poller/pull/132[#132]
```

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:21]: (Adding space to your suggestion so I can commit them all in one go from here)
```suggestion
The `bin/logstash-plugin uninstall <plugin>` command works as expected, successfully uninstalling the specified plugin https://github.com/elastic/logstash/pull/13823[#13823]
```

**Review by karenzone — APPROVED** [review]
Thanks for the fixes. LGTM

**Pull request merged**